### ISSUE TITLE
Фича: редактировать и удалить свои комментарии

### ISSUE THREAD

**Issue by hudochenkov** [vas3k/vas3k.club#276]
Добавить возможность удалить свой комментарий.

Добавить редактирование своих комментариев. Клуб может же быть лучше, чем твиттер :)

**Comment by vas3k** [comment]
А вот такого у тебя нет? 

<img width="238" alt="Screenshot 2020-06-13 at 19 12 19" src="https://user-images.githubusercontent.com/176344/84574886-d30acc80-ada9-11ea-876a-ef0b21caa0fd.png">

**Comment by hudochenkov** [comment]
Нету :(

<img width="741" alt="Screen Shot 2020-06-13 at 19 15 37" src="https://user-images.githubusercontent.com/654597/84574964-46144300-adaa-11ea-994b-b6be6b8bc3ca.png">

https://vas3k.club/post/1494/#comment-6e5338f5-2cd0-4a14-947a-acd93be4e46f

**Comment by vas3k** [comment]
Комментарии можно удалять или редактировать только первые 3 часа. Потом они становятся вечными. Этот создан 3 дня назад.

**Comment by hudochenkov** [comment]
Аааа! Это объясняет. Заметил, что в батле я вижу эти иконки. Они, наверное, показываются только в комментах первого уровня, а в реплаях их нет.

**Comment by Konfuze** [comment]
В реплаях тоже есть, по ховеру:
![image](https://user-images.githubusercontent.com/131331/84579285-39f1ab00-add5-11ea-8a3c-ca0fa6907e7a.png)

**Comment by hudochenkov** [comment]
Из кода теперь ясно.

Обычный коммент:

https://github.com/vas3k/vas3k.club/blob/f2707099359ac53d989e178f832b416e0420bcf9/frontend/html/comments/types/normal.html#L45-L51

Реплай проверяет `is_editable`:

https://github.com/vas3k/vas3k.club/blob/f2707099359ac53d989e178f832b416e0420bcf9/frontend/html/comments/types/reply.html#L32-L37

`is_editable` как раз то,  про что Вастрик говорил выше.

https://github.com/vas3k/vas3k.club/blob/f2707099359ac53d989e178f832b416e0420bcf9/comments/models.py#L97-L98

Я бы сделал поведение для всех комментариев одинаковым, или использовать везде `is_editable` или нет. Чтобы UI был предсказуемым и консистентным.

**Comment by vas3k** [comment]
Да, эта функция была придумана позже, так что хорошо было бы везде на нее переехать. Можешь сделать :)

**Comment by Takheer** [comment]
Имхо это слишком грубая обратная связь, я даже на секунду поверил, что это я виноват и что-то делаю не так))
![image](https://user-images.githubusercontent.com/11667674/110615540-592b3680-81b5-11eb-8d9e-bfa036369e78.png)
Достаточно просто глушить кнопку и при наведении показывать тултип с объяснением, почему нельзя редактировать коммент
